dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/arquillian/arquillianexamples/TestPropertiesFactory.java

```diff
@@ -0,0 +1,20 @@
+package edu.harvard.iq.dataverse.arquillian.arquillianexamples;
+
+import edu.harvard.iq.dataverse.settings.FileSettingLocations;
+import edu.harvard.iq.dataverse.settings.FileSettingLocationsFactory;
+
+import javax.enterprise.inject.Produces;
+import javax.enterprise.inject.Specializes;
+
+public class TestPropertiesFactory extends FileSettingLocationsFactory {
+
+    @Override
+    @Produces @Specializes
+    public FileSettingLocations buildSettingLocations() {
+        FileSettingLocations locations = super.buildSettingLocations();
```

Comment:
Maybe we could already get rid of loading `{user.home}/.dataverse/dataverse.properties` in test?. This would require to not use `super.buildSettingLocations` but initializing whole FileSettingLocations in this method